en/flight_modes/takeoff.md

```diff
@@ -31,47 +31,83 @@ Parameter | Description
 <span id="COM_RC_STICK_OV"></span>[COM_RC_STICK_OV](../advanced_config/parameter_reference.md#COM_RC_STICK_OV) | The amount of stick movement that causes a transition to [Position mode](../flight_modes/position_mc.md) (if [COM_RC_OVERRIDE](#COM_RC_OVERRIDE) is enabled).
 
 <span id="fixed_wing"></span>
-## Fixed Wing (FW)
+## Fixed-wing (FW)
+
+The aircraft takes off in the current direction (heading) using either *catapult/hand-launch mode* or *runway takeoff mode*.
+The mode defaults to catapult/hand launch, but can be set to runway takeoff by setting [RWTO_TKOFF](#RWTO_TKOFF) to 1.
+
+There are generally two ways to start an automatic takeoff on fixed-wing vehicles: either by planning a mission takeoff and starting the mission, or by switching to the Takeoff flight mode and arm the vehicle.
+Both methods lead to very similar behavior, with the difference being that in a mission takeoff, the user can specify the takeoff course (vehicle will track this course after being clear of the runway) and takeoff altitude (vehicle will climb to this altitude while staying on the takeoff course).
+For the non-mission takeoff, the course is set to the vehicle heading on arming, and the altitude is set to [MIS_TAKEOFF_ALT](#MIS_TAKEOFF_ALT).

```

Comment:
Does this mean on reaching the altitude the vehicle will continue forever on the arming-set course in Takeoff mode? Or will it switch to some other mode on reaching the altitude (e.g. loiter/hold)?

Reply:
I see below it switches to hold. We should say that at this level.

I also "wonder" if we should pull the mission-mode takeoff information here into a note or its own section. I.e. I think this is useful to state, but we should try keep this whole doc as much about FW takeoff mode as possible and therefore keep the mission mode comparisons as an aside.

Reply:
>Does this mean on reaching the altitude the vehicle will continue forever on the arming-set course in Takeoff mode? Or will it switch to some other mode on reaching the altitude (e.g. loiter/hold)?

It will switch to Hold and circle around Home (Takeoff without mission), proceed with the mission otherwise

Reply:
Yes please feel free to add the info about Hold here.

Separating all the Mission-takeoff related things to a note sounds reasonable to me indeed. We anyway should add some very rough bullet points on how to plan a Mission Takeoff also to the Mission section, and then reference to the docs here (like "place a Takeoff item in the direction of the runway, and set the altitude higher than obstacles in close proximity".